docs/internals/howto-release-django.txt

```diff
@@ -56,7 +56,7 @@ You'll need a few things before getting started:
 
 * An install of some required Python packages:
 
-  .. code-block:: bash
+  .. code-block:: console
```

Comment:
I'd switch to the `.. highlight:: console` style in this file as well and then fix the current `::` code sections to be console sections (e.g. prepand `$ ` to the lines)